auth-web/src/stores/business.ts

```diff
@@ -254,7 +254,16 @@ export const useBusinessStore = defineStore('business', () => {
   }
 
   async function loadBusiness () {
-    const businessIdentifier = ConfigHelper.getFromSession(SessionStorageKeys.BusinessIdentifierKey)
+    let businessIdentifier = ConfigHelper.getFromSession(SessionStorageKeys.BusinessIdentifierKey)
+    // If we don't find anything in SessionStorage, try to grab it from the URL
+    if (businessIdentifier === null) {
+      const queryString = window.location.search
+      const urlParams = new URLSearchParams(queryString)
+      if (urlParams.get('businessid')) {
+        businessIdentifier = urlParams.get('businessid')
+      }
+    }
```

Comment:
You can get the URL query parameters from the `$route` object.

Example: https://github.com/bcgov/business-filings-ui/blob/0955745de0fcb6c0fe16d5b99e84bd94347c3505/src/App.vue#L451

Reply:
... Except that this code is in the store, which doesn't have access to $route.